### DISCUSSION TITLE
Support for PyCharm

### DISCUSSION THREAD

**Discussion by vrdmr** [Azure/azure-functions-python-worker#970] [General]
Looking to get some data if users want PyCharm integration(mainly authoring, debugging, and publishing) for Functions Apps. Please thumbs up or comment in this thread to let us know your thoughts on this.

Thanks!

**Comment by wolfgangas** [comment]
Debugging should be the top priority. I wonder why there is no Azure Toolkit for PyCharm. For all other JetBrains IDEs it is available (although with rather mediocre reviews).
